packages/utils/telemetry-utils/src/test/errorLogging.spec.ts

```diff
@@ -614,14 +632,32 @@ describe("normalizeError", () => {
                 assert.equal(expected.stack, "<<stack from input>>", "<<stack from input>> hint should be used if using stack from input error object");
                 expected.withExpectedTelemetryProps({ stack: inputStack });
             }
-
-            assert.deepStrictEqual(actual.getTelemetryProperties(), expected.expectedTelemetryProps, "telemetry props should match");
         }
         for (const annotationCase of Object.keys(annotationCases)) {
             const annotations = annotationCases[annotationCase];
+            let doneOnceForThisAnnotationCase = false;
             for (const caseName of Object.keys(untrustedInputs)) {
                 const getTestCase = untrustedInputs[caseName];
-                it(`Normalize untrusted error: ${caseName} (${annotationCase})`, () => {
+                if (!doneOnceForThisAnnotationCase) {
+                    doneOnceForThisAnnotationCase = true;
+                    // Each test case only differs by what stack/error are.  Test the rest only once per annotation case.
+                    it(`Normalize untrusted error full validation: (${annotationCase})`, () => {
+                        // Arrange
```

Comment:
#nit not a big fan of the `Arrange`/`Act' comments. I feel they're sort of redundant.

Reply:
Yeah, it can be.  I think I like the "Act" block most of all because it draws attention to what behavior is actually under test.  But 🤷‍♂️